pkgs/applications/misc/pdfstudioviewer/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv,
+  lib,
+  makeWrapper,
+  fetchurl,
+  dpkg,
+  makeDesktopItem,
+  copyDesktopItems,
+  autoPatchelfHook,
+  gst_all_1,
+  sane-backends,
+  xorg,
+  gnome2,
+  alsa-lib,
+  libgccjit,
+  jdk11
+  }:
```

Comment:
Please run `nixpkgs-fmt` on the entire file

Reply:
Done